agent/src/beerocks/slave/test/1905_beacon_query_to_vs_test.cpp

```diff
@@ -0,0 +1,131 @@
+
+#include "../1905_beacon_query_to_vs.h"
+#include <bcl/network/network_utils.h>
+#include <tlvf/CmduMessageTx.h>
+
+constexpr const int RET_OK          = 0;
+constexpr const int RET_NO_BEEROCKS = -1;
+constexpr const int RET_NO_1905     = -2;
+
+std::ostream &operator<<(std::ostream &o, const beerocks_message::sBeaconRequest11k &request)
+{
+    // NOTE: not all fields are printed
+    // TODO: complete all fields
+
+    o << "\n"
+      << "measurement_mode:         " << +request.measurement_mode << "\n"
+      << "channel:                  "
+      << "0x" << std::hex << +request.channel << "\n"
+      << "op_class:                 "
+      << "0x" << std::hex << request.op_class << "\n"
+      << "repeats:                  " << request.repeats << "\n"
+      << "rand_ival:                " << request.rand_ival << "\n"
+      << "duration:                 " << request.duration << "\n"
+      << "sta mac:                  " << request.sta_mac << "\n"
+      << "bssid:                    " << request.bssid << "\n"
+      << "parallel:                 " << +request.parallel << "\n"
+      << "enable:                   " << +request.enable << "\n"
+      << "request:                  " << +request.request << "\n"
+      << "report:                   " << +request.report << "\n"
+      << "mandatory_duration:       " << +request.mandatory_duration << "\n"
+      << "expected_reports_count:   " << +request.expected_reports_count << "\n"
+      << "use_optional_ssid:        " << +request.use_optional_ssid << "\n"
+      << "ssid:                     " << request.ssid << "\n"
+      << "use_optional_ap_ch_report:" << +request.use_optional_ap_ch_report << "\n";
+
+    for (int idx = 0; idx < request.use_optional_ap_ch_report; ++idx) {
+        o << "channel[" << idx << "]:    " << std::dec << +request.ap_ch_report[idx] << "\n";
+    }
+
+    return o;
+}
+
+std::ostream &operator<<(std::ostream &o,
+                         beerocks_message::cACTION_MONITOR_CLIENT_BEACON_11K_REQUEST &msg)
+{
+    // currently we don't print anything except for the params field
+    // we might want to print / verify the action op (currently hard coded in the class)
+    o << "ACTION_MONITOR_CLIENT_BEACON_11K_REQUEST:"
+      << "\n-----------------------------------------\n"
```

Comment:
Some static analyzers would complain because you are using the same literal string more than once.